### PR TITLE
ci: enable github workflow on each pull request (with github comments)

### PR DESCRIPTION
**Pull request by mostafakamar2308**
It will need to be tested, as I cannot test this on my local environment.
Thank you for your great work 🚀

### CODE REVIEW

**Comment by neuodev** [comment]
Please fix the broken pipeline @mostafakamar2308

**Comment by mostafakamar2308** [comment]
I resolved every TS conflict.
The rest of the errors in this image:
image

I think it's due to cached versions of the packages. As I built nova locally and it was built successfully.

File: pnpm-lock.yaml
Comment (neuodev) [inline pnpm-lock.yaml:1]: I don't understand why there is so much deletion in this file. This file should no include any changes.

**Comment by neuodev** [comment]
Please rebase you branch on top of the latest master @mostafakamar2308!

**Comment by neuodev** [comment]
msg

**Comment by neuodev** [comment]
🚀 Nova: https://https://example.com
🚀 Luna: https://https://example.com
🚀 Nebula: https://https://example.com

**Comment by neuodev** [comment]
🚀 Nova: https://nova-8ky4k2a7a-litespace-cores-projects.vercel.app
🚀 Luna: https://luna-c5l64myjp-litespace-cores-projects.vercel.app
🚀 Nebula: https://nebula-bdxc878yf-litespace-cores-projects.vercel.app

**Comment by neuodev** [comment]
🚀 Nova: https://nova-7ai1bkxy1-litespace-cores-projects.vercel.app
🚀 Luna: https://luna-7lo3w0i9r-litespace-cores-projects.vercel.app
🚀 Nebula: https://nebula-hrx2zrl47-litespace-cores-projects.vercel.app

**Review by neuodev — APPROVED** [review]

**Comment by neuodev** [comment]
🚀 Nova: https://nova-6pjd6xd24-litespace-cores-projects.vercel.app
🚀 Luna: https://luna-3mez5mhmt-litespace-cores-projects.vercel.app
🚀 Nebula: https://nebula-dfdgpgk7q-litespace-cores-projects.vercel.app

**Pull request merged**